src/main/bash/sdkman-env.sh

```diff
@@ -73,16 +52,36 @@ function __sdkman_generate_sdkmanrc() {
 	__sdkman_echo_green "$sdkmanrc created."
 }
 
-function __sdkman_is_blank_line() {
-	[[ -z "$1" ]]
-}
+function __sdkman_env_each_line() {
```

Comment:
Should this be called something like `__sdkman_env_each_candidate` instead? Maybe I misunderstood, but won't this function process the entire file for either the `install` or `use` subcommands? The loop actual looping of individual lines happening from line 67 onward.

Reply:
Yes exactly, thanks!